### ISSUE TITLE
Error trying to share notebook

### ISSUE THREAD

**Issue by lpadula** [laurent22/joplin#6188]
## Environment

Joplin version: 2.6.10
Joplin server version: 2.7.4
Platform: Windows
OS specifics: 11

## Steps to reproduce

1. Right click on notebook.
2. Click on "share notebook".
3. Add joplin user email (previously added on Joplin server).

Then got error: No such folder: 167a9ffafb9a4e7b8ebe22650eae16b9

## Describe what you expected to happen

Share notebook.

**Comment by laurent22** [comment]
I think you need to sync first?

**Comment by lpadula** [comment]
The notebook is sync:

![image](https://user-images.githubusercontent.com/1341504/155398939-979cad14-76fd-4e79-a23e-a45d0669155a.png)

**Comment by lpadula** [comment]
The notebook is sync:

![image](https://user-images.githubusercontent.com/1341504/155399192-23562024-8937-4ce1-8bb5-eccfd00dadf8.png)

**Comment by laurent22** [comment]
I cannot replicate this. Do you have any error in the log during sync? https://joplinapp.org/debugging

**Comment by lpadula** [comment]
Yes

![image](https://user-images.githubusercontent.com/1341504/159125437-2ac25f38-5847-44ce-bf20-87a703d2748a.png)

**Comment by lpadula** [comment]
I forgot to mention, im using this new version of Joplin:

Joplin 2.7.15 (prod, win32)

Client ID: f64b8449565f49738cbd92b9a9ad399f
Sync Version: 3
Profile Version: 41
Keychain Supported: Yes

Revision: 8352e23

**Comment by laurent22** [comment]
If you sync with another device, does this folder you want to share show up on the other device?

**Comment by lpadula** [comment]
Yes, but is not a problem with this one, im getting the error message on any folder.

**Comment by laurent22** [comment]
Normally the only way it would display this error is if the folder has not been synced to the Joplin Server. If you go under Help => Sync Status, there should be a line: `Folder: x/y`. What does it say for you?

**Comment by lpadula** [comment]
All notes are synced:

![image](https://user-images.githubusercontent.com/1341504/159143343-596da1fc-2815-4e20-a9f9-83c5a38e447a.png)

**Comment by kgadek** [comment]
Had the same issue, I've been able to fix this by using "Re-upload local data to sync target".
